auth/providers/azure/graph/msi_tokenprovider_test.go

```diff
@@ -28,9 +28,10 @@ func TestMSITokenProvider(t *testing.T) {
 	const (
 		inputAccessToken    = "inputAccessToken"
 		msiAccessToken      = "msiAccessToken"
-		tokenResponse       = `{"token_type":"Bearer","expires_in":"3599","access_token":"%s"}`
+		tokenResponse       = `{"access_token":"%s","expires_in":"86700","refresh_token":"","expires_on":"1732881796","not_before":"%d","resource":"https://management.azure.com","token_type":"Bearer"}`
```

Comment:
I think you put the `%d` on the wrong property?

Reply:
Nice catch thanks!